buildSrc/src/main/java/org/springframework/boot/build/context/properties/DocumentConfigurationProperties.java

```diff
@@ -81,8 +81,9 @@ void documentConfigurationProperties() throws IOException {
 						"spring.jooq", "spring.jdbc", "spring.jpa", "spring.r2dbc")
 				.addOverride("spring.datasource.dbcp2", "Commons DBCP2 specific settings")
 				.addOverride("spring.datasource.tomcat", "Tomcat datasource specific settings")
-				.addOverride("spring.datasource.hikari", "Hikari specific settings").addSection("transaction")
-				.withKeyPrefixes("spring.jta", "spring.transaction").addSection("integration")
+				.addOverride("spring.datasource.hikari",
+						"Hikari specific settings bound to an instance of Hikari datasource")
```

Comment:
Rather than `Hikari datasource`, could you please change this to `` `HikariDataSource` `` (including the back-ticks) so that it's clearer that we're referring to the class named HikariDataSource?

Reply:
Just made the change. Thanks for the suggestion.